Tools-Override/crossgen.sh

```diff
@@ -53,13 +53,15 @@ crossgen_everything()
 crossgen_single()
 {
     __file=$1
-    $__crossgen /Platform_Assemblies_Paths $__sharedFxDir:$__toolsDir /nologo /MissingDependenciesOK /ReadyToRun $__file > /dev/null
-    if [ $? -eq 0 ]; then
-      __outname="${__file/.dll/.ni.dll}"
-      __outname="${__outname/.exe/.ni.exe}"
-      echo "$__file -> $__outname"
-    else
-      echo "Unable to successfully compile $__file"
+    if [[ $__file != *.ni.dll && $__file != *.ni.exe ]]; then
+	$__crossgen /Platform_Assemblies_Paths $__sharedFxDir:$__toolsDir /nologo /MissingDependenciesOK /ReadyToRun $__file > /dev/null
```

Comment:
I did this on my OSX machine and I don't have my .emacs so the tabs made their way in.  I'll fix this.